pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2227,13 +2227,14 @@
         "4.5.0",
         "4.5.1",
         "4.5.2",
-        "4.5.3"
+        "4.5.3",
+        "4.5.4"
       ],
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.5.1",
-        "4.0.0.1": "4.5.2",
-        "4.0.0.2": "4.5.3"
+        "4.0.0.1": "4.5.3",
```

Comment:
Was it intentional to change the mapping for 4.0.0.1?  I see we shipped that version both in 4.5.2 and 4.5.3 (no assembly version change in this one).

Reply:
> Was it intentional to change the mapping for 4.0.0.1? 

yes just wanted to keep the recent one. Yes we shipped the same assembly in 4.5.2 & 4.5.3 which was like in 2018
we incremented the assembly version for 4.5.4 because it contains netfx assets.

Reply:
Ok, so long as we're being consistent here with how we handle it for all the other packages that @joperezr updated.